doc/source/cluster/getting-started.rst

```diff
@@ -42,34 +35,35 @@ Where to go from here?
     Understand the key concepts and main ways of interacting with a Ray cluster.
 
     +++
-    .. link-button:: cluster-key-concepts-under-construction
+    .. link-button:: cluster-key-concepts
         :type: ref
         :text: Learn Key Concepts
         :classes: btn-outline-info btn-block
 
     ---
 
-    **I want to run Ray on a cloud provider**
+    **I want to run Ray on Kubernetes**
     ^^^
-    Take a sample application designed to run on a laptop and scale it up in the
-    cloud. Access to an AWS or GCP account is required.
+    Deploy a Ray application to a Kubernetes cluster. You can run the tutorial on a
+    Kubernetes cluster or on your laptop via KinD.
 
     +++
-    .. link-button:: ref-cluster-quick-start-vms-under-construction
+    .. link-button:: kuberay-quickstart
         :type: ref
-        :text: Getting Started with Ray Clusters on VMs
+        :text: Get Started with Ray on Kubernetes
         :classes: btn-outline-info btn-block
+
     ---
 
-    **I want to run Ray on Kubernetes**
+    **I want to run Ray on a cloud provider**
     ^^^
-    Deploy a Ray application to a Kubernetes cluster. You can run the tutorial on a
-    remote Kubernetes cluster or on your laptop via KinD.
+    Take a sample application designed to run on a laptop and scale it up in the
+    cloud. Access to an AWS or GCP account is required.
 
     +++
-    .. link-button:: kuberay-quickstart
+    .. link-button:: vm-cluster-quick-start
         :type: ref
-        :text: Getting Started with Ray on Kubernetes
+        :text: Get Started with Ray on VMs
         :classes: btn-outline-info btn-block
 
```

Comment:
Forgot to do this earlier: add a card like "I want to run my application on an existing Ray cluster" and link to jobs?